src/Webinex.Calendar/Filters/EventFiltersProvider.cs

```diff
@@ -54,20 +54,9 @@ public Expression<Func<EventRow<TData>, bool>> Create()
         if (!State)
```

Comment:
get back data filter, it's not applied on second OR clause

Reply:
fixed